js/views/PageNav.js

```diff
@@ -485,11 +488,16 @@ export default class extends BaseVw {
     });
     this.$('.js-connManagementContainer').append(this.pageNavServersMenu.render().el);
 
+    if (this.addressBarIndicators) this.addressBarIndicators.remove();
+    this.addressBarIndicators = new AddressBarIndicators();
+    this.$('.js-addressBarIndicatorsContainer').replaceWith(this.addressBarIndicators.render().el);
```

Comment:
Do you want  to user replaceWith here? I don't see an issue with it, but usually we use append, since the remove will get rid of any previous content in the container.

Reply:
Using .append() creates a double/redundant container.
Using .html() also creates a double/redundant container.

<img width="336" alt="screen shot 2018-06-12 at 6 12 45 pm" src="https://user-images.githubusercontent.com/392798/41320100-679ec060-6e6c-11e8-9c55-4070b6ba1ad3.png">

Using replaceWith() does not. Thoughts? 

<img width="359" alt="screen shot 2018-06-12 at 6 12 14 pm" src="https://user-images.githubusercontent.com/392798/41320102-69d5bf46-6e6c-11e8-9ece-95110da9bbd4.png">

Reply:
Note: I have the images reversed above.

Reply:
Not sure if we ever do re-render this view, but just in case, you also want to pass over the previous state for the addressBarIndicators component, otherwise it's lost.

```
let initialAddressBarState = {};
if (this.addressBarIndicators) {
  this.addressBarIndicators.remove();
  initialAddressBarState = this.addressBarIndicators.getState();
}
this.addressBarIndicators = this.createChild(AddressBarIndicators, {
  initialState: initialAddressBarState,
});
```

(just another example of Backbone being really bare-bones leaving us to manage stuff like this)